ui/client/package.json

```diff
@@ -123,7 +123,10 @@
     "@types/react-custom-scrollbars": "^4.0.6",
     "@types/react-dom": "^16.9.5",
     "@types/react-redux": "^7.1.7",
-    "@types/react-router-dom": "^5.1.3",
+    "@types/react-router-dom": "~5.1.5",
+    "@types/react-select": "^3.0.13",
+    "@types/react-transition-group": "^4.4.0",
+    "@types/reactable": "file:local-types/reactable",
```

Comment:
@arkadius, this type module is replaced by a local copy to avoid repairing the original